lib/Rex/Service/OpenBSD.pm

```diff
@@ -24,34 +24,17 @@ sub new {
   bless( $self, $proto );
 
   $self->{commands} = {
-    start   => '/etc/rc.d/%s start',
-    restart => '/etc/rc.d/%s restart',
-    stop    => '/etc/rc.d/%s stop',
-    reload  => '/etc/rc.d/%s reload',
-    status  => '/etc/rc.d/%s status',
-    action  => '/etc/rc.d/%s %s',
+    start        => '/usr/sbin/rcctl start %s',
+    restart      => '/usr/sbin/rcctl restart %s',
+    stop         => '/usr/sbin/rcctl stop %s',
+    reload       => '/usr/sbin/rcctl reload %s',
+    status       => '/usr/sbin/rcctl check %s',
```

Comment:
I don't think that would be useful to map a check to rcctl check because it would break the compatibility of a Rexfile that we would use on another system.